pkg/actions/lua/lakefs/catalogexport/table_extractor.lua

```diff
@@ -0,0 +1,52 @@
+local pathlib = require("path")
+local strings = require("strings")
+local yaml = require("encoding/yaml")
+local common = require("lakefs/catalogexport/common")
+local hive = require("lakefs/catalogexport/hive")
+
+local LAKEFS_TABLES_BASE = "_lakefs_tables/"
+
+function _is_table_obj(entry, tables_base)
```

Comment:
why _ in the func prefix?

Reply:
private

Reply:
you are not exporting it - it is private

Reply:
👍